pkg/parser/ast/misc.go

```diff
@@ -556,6 +556,7 @@ type Prepared struct {
 	StmtType      string
 	Params        []ParamMarkerExpr
 	SchemaVersion int64
+	RelateVersion map[int64]uint64
```

Comment:
Add some comments for this field.
And why the SchemaVersion is not enough for the cache key?